### PR TITLE
Release 1.11.10.1 - Django 1.11.10

### PR DESCRIPTION
**Pull request by filwaitman**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@ CHANGELOG
 =========
 
 
+1.11.10 (unreleased)
```
Comment (czpython) [inline CHANGELOG.rst]: missing today's date

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,12 @@ CHANGELOG
...
+---------------------
+
+* Update to Django 1.11.10
```
Comment (czpython) [inline CHANGELOG.rst]: more details please.
https://github.com/aldryn/aldryn-django/pull/93/files

File: aldryn_django/__init__.py [hunk 3]
```diff
@@ -1 +1 @@
-__version__ = '1.11.9.1'
+__version__ = '1.11.10'
```
Comment (czpython) [inline aldryn_django/__init__.py]: the format is four numbers.
so 1.11.10.1

**Review by czpython — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (filwaitman) [reply CHANGELOG.rst]: I'll add the date just before closing PR in order to avoid rework

File: aldryn_django/__init__.py (hunk 3 above)
Comment (filwaitman) [reply aldryn_django/__init__.py]: ok

**Review by czpython — APPROVED** [review]

**Pull request merged**